src/components/data/data-utils.js

```diff
@@ -496,7 +498,7 @@ export const EntityUploader = ({ onSuccess, onDismiss, namespace, name, entityTy
             icon('downloadRegular', { style: { size: 14, marginRight: '0.5rem' } }),
             'Download ',
             h(Link, {
-              href: 'https://storage.googleapis.com/terra-featured-workspaces/Table_templates/2-template_sample-table.tsv',
+              href: dataProvider.tsvFeatures.sampleTSVLink,
```

Comment:
the `Ajax().Metrics.captureEvent(Events.workspaceSampleTsvDownload` just below this will also need a tweak to capture WDS vs. Event Service. My PR to handle that is close; I'll share once it's ready!